ccsds/TC-Packet.acn

```diff
@@ -45,7 +45,7 @@ TC-PacketDataField <INTEGER:secondaryHeaderFlag> []
 TC-DataWithSecondaryHeader []
 {
     secondaryHeader [] {
-        tcPacketPusVersionNumber [],
+        tcPacketPusVersionNumber NULL [pattern '0010'B],
```

Comment:
we should either use or delete PacketVersionNumber structure. 

https://github.com/n7space/asn1-pusc-lib/blob/master/ccsds/PacketTypes.asn1#L38

I have a feeling, that it was supposed to go here but something was not working... hence the strange NULL in ASN.1...

Reply:
LOL, it has bad version: https://github.com/n7space/asn1-pusc-lib/blob/master/ccsds/PacketTypes.acn#L26

:D